modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/Credential.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.orchestrator.credential;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.eclipse.xpanse.modules.credential.AbstractCredentialInfo;
+import org.eclipse.xpanse.modules.credential.CredentialCenter;
+import org.eclipse.xpanse.modules.models.enums.Csp;
+import org.eclipse.xpanse.orchestrator.OrchestratorPlugin;
+import org.eclipse.xpanse.orchestrator.OrchestratorService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+/**
+ * The Credential class can get the credential capabilities of all plugins and pass it to the
+ * CredentialCenter.
+ */
+@Component
+public class Credential {
+
+    @Autowired
+    private CredentialCenter credentialCenter;
+    @Autowired
+    private OrchestratorService orchestratorService;
+
+    /**
+     * Get all AbstractCredentialInfo list by Csp.
+     *
+     * @return return credentialMap;
+     */
+    @Bean
```

Comment:
is It possible to remove this annotation?

Reply:
all right

Reply:
ok, i have removed.